Clew/FeedbackLogger.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  FeedbackLogger.swift
+//  Clew
+//
+//  Created by tad on 6/7/19.
+//  Copyright © 2019 OccamLab. All rights reserved.
+//
+
+//imports lobraries
+import Foundation
+import Firebase
+
+
+class FeedbackLogger {
+
+    //creates a reference to the feedback folder within firebase
```

Comment:
Triple slashes `///` are used in most files for inline comments. Worth changing to be consistent?